source/_integrations/zha.markdown

```diff
@@ -163,11 +163,12 @@ Note that the `otau_directory` setting is optional and can be used for any firmw
 
 ## Adding devices
 
-Go to the **Configuration** page and select the **ZHA** integration that was added by the configuration steps above.
+To add a new device:
 
-Click on **ADD DEVICES** to start a scan for new devices.
-
-Reset your Zigbee devices according to the device instructions provided by the manufacturer (e.g.,  turn on/off lights up to 10 times, switches usually have a reset button/pin).
+1. Go to the **Integrations** page, find the **Zigbee Home Automation** integration that was added by the configuration steps above, and select **Configure**.
+1. Click on the plus button at the bottom right corner to start a scan for new devices.
+1. Reset your Zigbee devices according to the device instructions provided by the manufacturer (e.g.,  turn on/off lights up to 10 times, switches usually have a reset button/pin). It might take a few seconds for the devices to appear. You can click on **Show logs** for a more verbose output.
+1. Once the device is found, it will appear in that page and will be automatically added to your devices. You can optionally change its name and add it to an area (you can change this later). You can search again to add another device, or you can go back to the list of added devices.
```

Comment:
```suggestion
1. Go to the **Integrations** page, find the **Zigbee Home Automation** integration that was added by the configuration steps above, and select **Configure**.
1. Click on the plus button at the bottom right corner to start a scan for new devices.
1. Reset your Zigbee devices according to the device instructions provided by the manufacturer (e.g., turn on/off lights up to 10 times, switches usually have a reset button/pin). It might take a few seconds for the devices to appear. You can click on **Show logs** for more verbose output.
1. Once the device is found, it will appear on that page and will be automatically added to your devices. You can optionally change its name and add it to an area (you can change this later). You can search again to add another device, or you can go back to the list of added devices.
```